<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TNINCHD115" failures="0" tests="2" name="com.qa.test.A" time="30.021" errors="1" timestamp="2025-04-14T13:49:30 IST" skipped="0">
  <testcase classname="com.qa.test.A" name="launchBrowser" time="28.668"/>
  <testcase classname="com.qa.test.A" name="endExtentReport" time="1.353">
    <error message="Cannot run program &quot;C:\Users\vikas.verma\eclipse-workspace\Ascent_One_Automation/allureReport.bat&quot;: CreateProcess error=2, The system cannot find the file specified" type="java.io.IOException">
      <![CDATA[java.io.IOException: Cannot run program "C:\Users\vikas.verma\eclipse-workspace\Ascent_One_Automation/allureReport.bat": CreateProcess error=2, The system cannot find the file specified
at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
at com.ascentone.qa.base.TestBase.runBatFile(TestBase.java:183)
at com.ascentone.qa.base.TestBase.gererateAllureReport(TestBase.java:281)
at com.ascentone.qa.base.TestBase.endExtentReport(TestBase.java:273)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:386)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
at java.base/java.lang.ProcessImpl.create(Native Method)
at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:506)
at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:159)
at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
... 21 more
]]>
    </error>
  </testcase> <!-- endExtentReport -->
</testsuite> <!-- com.qa.test.A -->
